website/blog/2021-08-12-tungsten.md

```diff
@@ -0,0 +1,321 @@
+---
+author: Vadim Chelyshov
+title: Metals v0.10.5 - Tungsten
+authorURL: https://twitter.com/_dos65
+authorImageURL: https://github.com/dos65.png
+---
+
+We're happy to announce the release of Metals v0.10.5, which contains a lot of fixes, adds Scala 3.0.1 support,
+and even brings one nice feature!
+
+<table>
+<tbody>
+  <tr>
+    <td>Commits since last release</td>
+    <td align="center">150</td>
+  </tr>
+  <tr>
+    <td>Merged PRs</td>
+    <td align="center">71</td>
+  </tr>
+    <tr>
+    <td>Contributors</td>
+    <td align="center">9</td>
+  </tr>
+  <tr>
+    <td>Closed issues</td>
+    <td align="center">18</td>
+  </tr>
+  <tr>
+    <td>New features</td>
+    <td align="center">1</td>
+  </tr>
+</tbody>
+</table>
+
+For full details: https://github.com/scalameta/metals/milestone/40?closed=1
+
+Metals is a language server for Scala that works with VS Code, Vim, Emacs,
+Sublime Text and Eclipse. Metals is developed at the
+[Scala Center](https://scala.epfl.ch/) and [VirtusLab](https://virtuslab.com)
+with the help from [Lunatech](https://lunatech.com) along with contributors from
+the community.
+
+## TL;DR
+
+Check out [https://scalameta.org/metals/](https://scalameta.org/metals/), and
+give Metals a try!
+
+- Add support for `textDocument/selectionRange`
+- [Scala 3] `Inferred Type` code action and other improvements
+- Formatter - `runner.dialect` support and `.scalafmt.conf` update suggestions
+- Dependency sources - search symbol references in the workspace
+
+## Add support for `textDocument/selectionRange`
+
+Due to the great work by [@ckipp01](https://github.com/ckipp01)  now Metals implements `textDocument/selectionRange` LSP-method.
+It allows you to easily select an expression and expand it using a single shortcut combination.
+
+![gif1](https://user-images.githubusercontent.com/13974112/121028775-605be380-c7a8-11eb-99f1-948ce0b730ab.gif)
+
+## [Scala3] `Inferred Type` code action and other improvements
+
+We keep working on implementing the same feature set  for Scala 3 as Scala 2 has.
+This release adds Inferred Type code action support.
+
+![gif2](https://i.imgur.com/GJGFOOy.gif)
+
+In addition, ignoring other small fixes, it worth mentioning that `Go to Definition also was improved.
```

Comment:
```suggestion
In addition to other small fixes, it's worth mentioning that the `Go to Definition` functionality was improved so that in situations where there are compilation issues, you're now able to better navigate to the definition site.
```

Then I'd just drop the following sentence.